Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic.yml

```diff
@@ -0,0 +1,1063 @@
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 93fe0ced-0cc0-4fe6-8d97-11b74243539b
+    type: title
+    task:
+      id: 93fe0ced-0cc0-4fe6-8d97-11b74243539b
+      version: -1
+      name: Failed Logon Search
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "73"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 370,
+          "y": 400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 079af6ed-0728-4152-8b8f-6713b79c85bc
+    type: title
+    task:
+      id: 079af6ed-0728-4152-8b8f-6713b79c85bc
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "48"
+      - "49"
+      - "51"
+      - "74"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": 10
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "28":
+    id: "28"
+    taskid: 85e91ccb-1565-48a5-8f56-e745b4b3d5a2
+    type: title
+    task:
+      id: 85e91ccb-1565-48a5-8f56-e745b4b3d5a2
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 710,
+          "y": 1680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: b632202b-40e8-41aa-8de5-b98ddef463b2
+    type: title
+    task:
+      id: b632202b-40e8-41aa-8de5-b98ddef463b2
+      version: -1
+      name: Threat Logs Search
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "32"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1600,
+          "y": 400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: b12fcd5a-2d12-49a3-8bb3-36035bd2a0e0
+    type: condition
+    task:
+      id: b12fcd5a-2d12-49a3-8bb3-36035bd2a0e0
+      version: -1
+      name: Is PAN-OS integration enabled?
```

Comment:
You dont have validation if a username provided, need to add

Reply:
Done.